frappe/docs/user/en/guides/integration/google_gsuite.md

```diff
@@ -0,0 +1,68 @@
+# Google GSuite
+
+You can create and attach Google GSuite Docs to your Documents using your predefined GSuite Templates.
+These Templates could use variables from Doctype that will be automatically filled.
+
+## Enable integration with Google Gsuite
+
+### Publish Google apps script
+
+1. Go to [https://script.google.com](https://script.google.com)
+1. Create a new Project. Click on **File > New > Project**
+1. Fill Code.gs the content of ***<bench dir>/apps/frappe/integrations/doctype/gsuite_settings/gsuite_settings.gs***
```

Comment:
Most erpnext users don't understand the need between a `git stash` and `git commit`. Asking users to change a file is a complete no no. For all integrations all you need is to update your API keys.

Can't you just include that file as a template? And replace the dynamic part?